.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - "checkout"
       - run: "sudo npm i -g npm && npm ci"
       - run: "bash ~/project/.circleci/create_netrc_file.sh"
-      - run: "git push https://git.heroku.com/origami-polyfill-service-int.git HEAD:master --force"
+      - run: "git push https://git.heroku.com/origami-polyfill-service-int.git HEAD^{}:refs/heads/master --force"
```

Comment:
This looks reasonable, approved! 

_except I don't don't what the purpose of `^{}` is or when writing a non-tag object is a problem?_ 😳

Reply:
I think I don't need to do this actually, I think Heroku can instead reset their git repo and that should resolve the issue.